sdk/tables/Azure.Data.Tables/samples/Sample2CreateDeleteEntities.md

```diff
@@ -0,0 +1,95 @@
+# Create and Delete Entities
+This sample demonstrates how to add and delete entities. To get started, you'll need access to either a Storage or Cosmos DB account.
```

Comment:
Maybe say `add and delete entities to a table`

Reply:
Yeah, I wasn't sure if I should go ahead and do that since the current SDK has `CreateEntity` as the way to add the entity to the service, but I thiiink it's supposed to change based on Krzysztof's feedback.

Reply:
ohhh.. didn't know that. you could leave it as is, and whenever the new implementation comes, someone can change it